client_with_ssl.js

```diff
@@ -73,12 +72,16 @@ async function fetchCities(client) {
 
     console.log("OK.");
     console.log("--Results of 'SELECT __key, this FROM cities'");
-    printf("| %4s | %20s | %20s | %15s |\n", "id", "country", "city", "population");
+    sprintf("| %4$s | %20$s | %20$s | %15$s |\n", "id", "country", "city", "population");
+
+    // NodeJS client does lazy deserialization. In order to update schema table on the client,
+    // it's required to get map. 
+    const cities = await client.getMap("cities");
```

Comment:
a map.get() is required, not getMap

Reply:
Fixed at https://github.com/hazelcast/hazelcast-cloud-nodejs-sample-client/pull/20/commits/d1a77886785820dc196122075974dd89a2a81494